libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/TripSessionLocationEngine.kt

```diff
@@ -78,6 +78,9 @@ internal class TripSessionLocationEngine constructor(
     }
 
     fun stopLocationUpdates() {
+        if (isReplayEnabled) {
```

Comment:
The only side-effect I see here is if someone uses this class directly and subscribes to location updates on their own, not caring about Mapbox trip sessions, in some cases they won't get last location asap (if we had last location from previous session), but they'll have to wait for a new one. 
I think this is acceptable, because:
A) This class is not designed to be used this way;
B) This isn't a breaking change behaviour-wise.